pkg/apis/scheduling/v1/helpers.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Do we need to change the copyright year?

Reply:
mmm am not sure, actually i was copy-pasting the versioned apis to a new pkg but the github insists it's a renaming 🤷🏻‍♂️